views/timeline.html

```diff
@@ -9,18 +9,18 @@
 			type="text/css"
 			href="https://Replit-2020.whippingdot.repl.co/static/style.css"
 		/>
-    <link
-    rel="shortcut icon"
-    href="favicon.ico"
-    type="image/x-icon"
-    />
+		<link
+			rel="shortcut icon"
+			href="https://Replit-2020.whippingdot.repl.co/static/favicon_but_better.ico"
+			type="image/x-icon"
+		/>
 	</head>
 	<body>
 		<header class="center">
 			<h1 class="big">For Repl.it</h1>
 			<h1>Share It</h1>
 			<div class="navbar">
```

Comment:
navbar class? so what is the `nav` element for?

Reply:
I didn't even know that was a thing...Anyway someone else did that, not me. Just leave it be, makes no difference. If you want you can change it.